src/foam/nanos/auth/Relationships.js

```diff
@@ -3,49 +3,58 @@
  * Copyright 2017 The FOAM Authors. All Rights Reserved.
  * http://www.apache.org/licenses/LICENSE-2.0
  */
+
 /*
- foam.RELATIONSHIP({
-   cardinality: '*:*',
-   sourceModel: 'foam.nanos.auth.Group',
-   targetModel: 'foam.nanos.auth.Permission',
-   forwardName: 'permissions',
-   inverseName: 'groups',
-   sourceProperty: {
-     hidden: true
-   },
-   targetProperty: {
-     hidden: true
-   }
- });
+foam.RELATIONSHIP({
+  cardinality: '*:*',
+  sourceModel: 'foam.nanos.auth.Group',
+  targetModel: 'foam.nanos.auth.Permission',
+  forwardName: 'permissions',
+  inverseName: 'groups',
+  sourceProperty: {
+    hidden: true
+  },
+  targetProperty: {
+    hidden: true
+  }
+});
 */
 
 
 /*
- foam.RELATIONSHIP({
-   cardinality: '*:*',
-   sourceModel: 'foam.nanos.auth.User',
-   targetModel: 'foam.nanos.auth.Group',
-   forwardName: 'groups',
-   inverseName: 'users',
-   sourceProperty: {
-     hidden: true
-   },
-   targetProperty: {
-     hidden: true
-   }
- });
+foam.RELATIONSHIP({
+  cardinality: '*:*',
+  sourceModel: 'foam.nanos.auth.User',
+  targetModel: 'foam.nanos.auth.Group',
+  forwardName: 'groups',
+  inverseName: 'users',
+  sourceProperty: {
+    hidden: true
+  },
+  targetProperty: {
+    hidden: true
+  }
+});
 */
 
- foam.RELATIONSHIP({
-   cardinality: '1:*',
-   sourceModel: 'foam.nanos.auth.Group',
-   targetModel: 'foam.nanos.auth.User',
-   forwardName: 'users',
-   inverseName: 'group',
-   sourceProperty: {
-     hidden: true
-   },
-   targetProperty: {
-     hidden: true
-   }
- });
+foam.RELATIONSHIP({
+  cardinality: '1:*',
+  sourceModel: 'foam.nanos.auth.Group',
+  targetModel: 'foam.nanos.auth.User',
+  forwardName: 'users',
+  inverseName: 'group',
+  sourceProperty: {
+    hidden: true
+  },
+  targetProperty: {
+    hidden: true
+  }
+});
+
+foam.RELATIONSHIP({
```

Comment:
This isn't going to work because Address doesn't have a primaryKey or a DAO. Instead it should be an FObjectArray. The factory could return an array of 7 Hours already setup for each day.

Reply:
Thanks. Done.